src/lando/dockerflow/decorators.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import time
+from functools import wraps
+
+# Logging is not yet implemented
```

Comment:
"not yet implemented" is a little inaccurate, since logging should just work out of the box. However, custom logging (i.e., in the way we intend for it to be in the long term, e.g. with mozlog, and where the log output is going, etc.) is not yet implemented.

Reply:
I'll remove the comment.